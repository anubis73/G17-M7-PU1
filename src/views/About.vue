<template>
  <div class="about">
    <h1>About Page</h1>
    <p>Esta es la página About de nuestra aplicación Vue.js</p>
  </div>
</template>

<script>
export default {
  name: 'AboutView'
}
</script>

<style scoped>
.about {
  padding: 20px;
}
</style>
