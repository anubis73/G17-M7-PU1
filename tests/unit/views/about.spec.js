import { shallowMount } from '@vue/test-utils'
import About from '@/views/About.vue'

describe('About.vue', () => {
  it('renders about page content', () => {
    const wrapper = shallowMount(About)
    expect(wrapper.find('h1').text()).toBe('About Page')
  })

  it('matches snapshot', () => {
    const wrapper = shallowMount(About)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
