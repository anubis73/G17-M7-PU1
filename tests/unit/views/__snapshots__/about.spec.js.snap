// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About.vue matches snapshot 1`] = `
<div class="about">
  <h1>About Page</h1>
  <p>Esta es la página About de nuestra aplicación Vue.js</p>
</div>
`;
